win/CS/HandBrakeWPF/ViewModels/ChaptersViewModel.cs

```diff
@@ -210,6 +221,57 @@ public void Import()
         }
 
         /// <summary>
+        /// Validates any imported chapter information against the currently detected chapter information in the 
+        /// source media. If validation fails then an error message is returned via the out parameter <see cref="validationErrorMessage"/>
+        /// </summary>
+        /// <param name="importedChapters">The list of imported chapter information</param>
+        /// <param name="validationErrorMessage">In case of a validation error this variable will hold 
+        ///                                      a detailed message that can be presented to the user</param>
+        /// <returns>True if there are no errors with imported chapters, false otherwise</returns>
+        private bool ValidateImportedChapters(Dictionary<int, Tuple<string, TimeSpan>> importedChapters, out string validationErrorMessage)
+        {
+            validationErrorMessage = null;
+
+            // If the number of chapters don't match, prompt for confirmation
+            if (importedChapters.Count != this.Task.ChapterNames.Count)
+            {
+                if (DialogResult.Yes !=
+                    MessageBox.Show(

```

Comment:
Just a quick note, I have a wrapper around message box in the error service (if i recall correctly).  Handy if I ever decide to write a unit test suite for the UI since it allows the mocking of UI responses.